packages/blade/src/components/NavigationBar/_decisions/decisions.md

```diff
@@ -0,0 +1,261 @@
+
+**Q:** So where will the product navigation items go? 
+**Ans:** The product navigation items will be moved to the bottom of the screen, but this will be part of a separate component called `BottomNavigation` which we will ship in phase 2.
+
+Mobile Navbar example:
+
+```jsx
+const Dashboard = () => {
+  const { isMobile } = useBreakpoint();
+
+  if (isMobile) {
+    return (
+      <NavigationBar>
+        <NavigationBarContent>
+          <Button icon={MenuIcon} variant="secondary" accessibilityLabel="Toggle side navigation" />
+          <Text>{currentPageTitle}</Text>
+        </NavigationBarContent>
+        <NavigationBarActions>
+          <SearchInput />
+          <Button icon={ActivityIcon} accessibilityLabel="View Ecosystem Health" />
+          <Button icon={AnnouncementIcon} accessibilityLabel="View Announcements" />
+          <Button icon={RayIcon} accessibilityLabel="Ask Ray" />
+          <Avatar onClick={openBottomSheet} />
+          <BottomSheet>
+            { /* Avatar BottomSheet Content */ }
+          </BottomSheet>
+        </NavigationBarActions>
+      </NavigationBar>
+    )
+  }
+
+  return (
+    // ... desktop navigation bar jsx
+  )
+}
+```
+
+## Accessibility
+
+- TabNav
+  - Tab items will have `role=link` instead of `tab` because they don't contain any related tab panels.
+  - Tab nav items will be a set of links that will be navigable using the keyboard.
+  - The selected tab will have an `aria-selected` attribute set to `true`.
+
+- NavigationBar
+  - The navigation bar will have `role=navigation` to indicate that it is a navigation landmark.
+  - NavigationBar sub components will only compose other components and will not have any ARIA roles or attributes.
+
+## References
+
+- [NavBar - NextUI](https://nextui.org/docs/components/navbar)
+- [AppBar - MUI](https://mui.com/material-ui/react-app-bar/)
+- [FullscreenNav - Shopify](https://polaris.shopify.com/components/navigation/fullscreen-bar)
+- [Navigation - Atlassian DS](https://atlassian.design/components/atlassian-navigation/examples)
+- [PageHeader - Primer](https://primer.style/components/page-header/react/draft)
+- [TabNav - Primer](https://primer.style/components/tab-nav/react/alpha)
+- [HeaderNav - Carbon DS](https://react.carbondesignsystem.com/?path=/story/components-ui-shell-header--header-w-navigation-and-actions)
+- [HeaderNav - BaseWeb DS / Uber](https://baseweb.design/components/header-navigation/#basic-link1)
+- [TopNav - Spectrum / Adobe](https://opensource.adobe.com/spectrum-web-components/components/top-nav/)
+
+## Open Questions
+
+### Dev
+
+1. How will the Menu component integrate with the BottomSheet?
+2. How will the TabNav component handle the chevron icon for the Menu component?
+3. Should we call this component `TopNav`/`TopNavigationBar` instead of `NavigationBar`?
```

Comment:
TopNav might help if we're also going to create BottomNav.

If we can have BottomNav inside of this itself, then `NavBar` might be good?

Reply:
Yeah BottomNav will be a thing, so maybe TopNav is a better option. Can get opinion of others too.